euphoria-core/src/main/java/cz/seznam/euphoria/core/client/dataset/InputDataset.java

```diff
@@ -20,8 +20,12 @@
 import cz.seznam.euphoria.core.client.io.DataSink;
 import cz.seznam.euphoria.core.client.io.DataSource;
 import cz.seznam.euphoria.core.client.operator.Operator;
-import java.util.Collection;
+import cz.seznam.euphoria.core.client.operator.hint.OutputHint;
+
 import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
 
 /**
  * {@code PCollection} that is input of a {@code Flow}.
```

Comment:
Although this is not part of this PR, can we fix the comment here?

Reply:
fixed in #271